docs/service-introspection-onboarding.md

```diff
@@ -1,22 +1,42 @@
 # Onboard a Bedrock project to use Service Introspection
 
-If you have already followed the steps
-[here](https://github.com/microsoft/bedrock/tree/master/gitops) to setup the
-pipelines for a GitOps workflow in Bedrock, you may update your pipelines to
-send data to the Spektate storage, which will help you run the introspection
-tool on your services.
+Service Introspection shows information about a
+[Bedrock GitOps workflow](https://github.com/microsoft/bedrock/tree/master/gitops).
+
+Service introspection is used via the `spk deployment` commands. More
+information about the commands is available in the command reference
+[here](https://github.com/CatalystCode/spk/blob/master/docs/service-introspection.md).
+
+To use service introspection you first need to make sure you have the following
+pre-requisites.
 
 ## Pre-Requisites
 
-Service introspection tool needs an Azure storage account to store information
-about your pipelines and services.
+1. GitOps pipelines workflow in Bedrock. To setup the workflow, follow
+   [these](https://github.com/microsoft/bedrock/tree/master/gitops)
+   instructions.
+2. [Service introspection storage in Azure](#service-introspection-storage). See
+   below for instructions on how to create one.
+3. [Pipelies configuration](#pipelines-configuration)
+
+### Service introspection storage
+
+Service introspection tool needs a database to store the information about your
+pipelines and services. Currently, service introspection supports storage in the
+form of an Azure Storage table. Follow the steps below to create it.
+
+1. Create an Azure storage account:
+
+#### Option 1:
+
+Use the
+[`spk deployment onboard`](https://github.com/CatalystCode/spk/blob/master/docs/service-introspection.md#onboard)
+command.
 
-If you don't already have an Azure storage account you would like to use, use
-the `spk deployment onboard` command which will create a storage account in your
-subscription.
+#### Option 2:
 
-You may also create this storage account manually. You will need to have the
-following properties of this storage before proceeding:
+Create the account manually. You will need to have the following properties of
+this storage before proceeding:
```

Comment:
before proceeding as they are required to configure <add a link to Configure a variable group section>